config/general.go

```diff
@@ -11,14 +16,28 @@ type GeneralConfig struct {
 	Mouse              bool           `toml:"mouse"`
 	Timestamps         bool           `toml:"timestamps"`
 	Identify           IdentifyConfig `toml:"identify"`
+	DownloadLocation   string         `toml:"attachment_download_location"`
```

Comment:
Renaming this field as well as `toml` tag to `DownloadsDir` is preferable to be more specific.